pinot-controller/src/main/resources/app/components/Homepage/Operations/RebalanceServerStatusOp.tsx

```diff
@@ -63,7 +64,10 @@ export const RebalanceServerStatusOp = (
                 if (jobs.error) {
                     return;
                 }
-                setRebalanceServerJobs(jobs as RebalanceTableSegmentJobs)
+                const sortedJobs: RebalanceTableSegmentJob[] = Object.keys(jobs as RebalanceTableSegmentJobs)
+                    .map(jobId => jobs[jobId] as RebalanceTableSegmentJob)
+                    .sort((j1, j2) => j1.submissionTimeMs < j2.submissionTimeMs ? 1 : -1);
+                setRebalanceServerJobs(sortedJobs);
```

Comment:
do this inside the axios fetch function itself.

Reply:
ok!

Reply:
It's difficult to do inside `axios` function as the same axios function is being used by other code components and the axios returns an object with job information as key & object pairs. That's the reason why I'm flattening it here into an array and then sorting it